interaction_rating.go

```diff
@@ -21,6 +22,8 @@ func AddRating(userId string, itemId string, rating float64, opts ...RequestOpti
 	}
 }
 
+// DeleteRating deletes an existing rating specified by (userId, itemId, timestamp) from the database or all the
+// ratings with the given userId and itemId if timestamp is omitted.
```

Comment:
I suppose the timestamp for delete serves for the different delete API calls the same. But in other section this is not included. 

I think here is point where the user should refer to `request_options.go` or official API reference, where this info is given.